pkg/apis/workflow/register.go

```diff
@@ -23,38 +23,5 @@ const (
 	ClusterWorkflowTemplatePlural    string = "clusterworkflowtemplates"
 	ClusterWorkflowTemplateShortName string = "cwftmpl"
 	ClusterWorkflowTemplateFullName  string = ClusterWorkflowTemplatePlural + "." + Group
+	WorkflowEventBindingKind         string = "WorkflowEventBinding"
 )
-
-type CRD struct {
```

Comment:
I remembered we need to add new CRD for CRD generation. Do we not need it now?

Reply:
unused code